sms_sponsorship/__manifest__.py

```diff
@@ -12,11 +12,13 @@
     'name': 'Compassion SMS Sponsorships',
     'version': '10.0.0.2.0',
     'category': 'Other',
-    'author': 'Compassion CH',
+    'author': 'Compassion CH, Odoo Community Association (OCA)',
```

Comment:
Remove OCA (the above comment  # pylint: disable=C8101 should take care of the lint error)